### DISCUSSION TITLE
One repository has not been changed to use Codemagic/Github integration

### DISCUSSION THREAD

**Discussion by Douglal** [codemagic-ci-cd/codemagic-docs#2000] [Q&A]
### If you are looking for help, please confirm the following...

- [X] I confirm I have searched the [Docs](https://docs.codemagic.io), [Codemagic Sample Projects](https://github.com/codemagic-ci-cd/codemagic-sample-projects), and GitHub [Discussions](https://github.com/orgs/codemagic-ci-cd/discussions).
- [X] I confirm the software versions on my local computer match the ones in CI environment (e.g. Xcode version, Gradle version). See [here](https://docs.codemagic.io/troubleshooting/common-android-issues/#builds-work-locally-but-fail-on-codemagic)
- [X] I confirm I am able to do a fresh clone of the repository and build the project on my local computer.

### Which mobile framework are you using?

Flutter (Dart)

### Steps to reproduce

Codemagic CI/CD is set up in GitHub for "All repositories"
Correct GitHub URLs are set for all of my apps in Codemagic

### Expected results

All apps use Codemagic CI/CD with access to each app's private GitHub repository

### Actual results

One of my apps cannot access its GitHub repository from Codemagic. Error message:
  Cannot access the repository https://github.com/Douglal/BLISS
'Repository access settings' shows 'Authentication method: GitHub personal access token' and there seems to be no way to change this to use Codemagic CI/CD integration.
All other apps access their repositories correctly using just the Repository URL

**Comment by dtrdic** [comment]
Hello @Douglal, try adding that repo through "Other" option
![image](https://github.com/codemagic-ci-cd/codemagic-docs/assets/38918425/9da77882-9be6-418e-92a3-30c2626c4d07)

then just use http or ssh link to clone the repo
![image](https://github.com/codemagic-ci-cd/codemagic-docs/assets/38918425/89609fe6-8494-4eed-8e7d-ec1faf4f952d)

**Comment by Douglal** [comment]
Thank you David. I couldn't find how to do this for the existing app. So I just deleted the app in Codemagic and added it back which has worked fine.
